redash/models/__init__.py

```diff
@@ -549,6 +549,23 @@ def by_user(cls, user):
     def by_api_key(cls, api_key):
         return cls.query.filter(cls.api_key == api_key).one()
 
+    @classmethod
+    def past_scheduled_queries(cls):
+        now = utils.utcnow()
+        queries = (
+            Query.query
+            .options(joinedload(Query.latest_query_data).load_only('retrieved_at'))
+            .filter(Query.schedule.isnot(None))
+            .order_by(Query.id)
+        )
+        past_scheduled_queries = []
```

Comment:
Does it feel like a comprehension or filter would be more apt here? Not sure.

Reply:
Used filter here. Also, added another unit test for this method specifically